pkg/auditlog/storage/v2/admin_audit_log.go

```diff
@@ -54,22 +61,12 @@ func (s *adminAuditLogStorage) CreateAdminAuditLogs(ctx context.Context, auditLo
 		return nil
 	}
 	var query strings.Builder
-	query.WriteString(`
```

Comment:
To maintain consistency, we should do the following:

#### Storage

https://github.com/bucketeer-io/bucketeer/blob/main/pkg/account/storage/v2/account.go#L55-L67

#### Query file

https://github.com/bucketeer-io/bucketeer/blob/main/pkg/account/storage/v2/sql/account_v2/insert_account_v2.sql

We don't need to use strings.Builder.

Reply:
After thinking about it, since I need to insert multiple pieces of data here, I think it would be more efficient to use string.Builder to concatenate the required amount of strings. What do you think?
(The example account inserts one piece of data.)
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/auditlog/storage/v2/audit_log.go#L52